enhancements/console/dynamic-plugins.md

```diff
@@ -433,6 +433,33 @@ const VMHeading = () => {
 };
 ```
 
+For a 4.11 release a `console.openshift.io/default-i18next-namespace` annotation,
+is being introduced. It annotation will indicate whether the dynamic plugin containes 
+localization resources, or not. If the annotation value is set to `true`, the localization
+resources, from the namespace named after the dnyamic plugin(eg. `plugin__kubevirt`),
+are loaded. If the annotation is set to `false`, or it is missing on the dynamic plugin CR,
+the localization resources are not loaded.
+Note: All API versions of Dynamic plugins CRD are checked for this annotation.
+
+Prior to 4.11 release, localization resources are being loaded by default. In case these
+resources are not present in the dynamic plugin, the initial console load will be slowed 
+down. For more info check [BZ#2015654](https://bugzilla.redhat.com/show_bug.cgi?id=2015654)
+
+For `v1` API version an additional array field is introduced, for listing all the available
+i18n namespaces that given dynamic plugin contains. Only namespaces listed in the spec will we loaded.
+
+```yaml
+apiVersion: console.openshift.io/v1alpha1
+kind: ConsolePlugin
+metadata:
+  name: acm
+spec:
+  displayName: 'Advanced Cluster Management'
+  i18n:
+    namespaces:
+      - plugin__acm
```

Comment:
We'll need to decide if (1) we want to support multiple namespaces and (2) if we should ensure that namespace names are unique (i.e. by including the plugin name). It would be good to get stakeholder feedback and discuss at the next sync.

cc @christianvogt @bipuladh

Reply:
An argument for multiple namespaces is that a shared library is being used and manages it's own namespace, then the host plugin must register the shared library namespace with it's plugin because the console must relay the call to the correct service hosting the locales for that shared library.

This however has potential to clash with other plugins that may be using the same library but different versions.

How can the console ensure that the shared library localization requests go to the correct service if multiple plugins use the same shared library with the same namespace?

The onus is also then on a plugin to prevent clashing with other plugins. This is problematic. The uniqueness of plugin is in its name and nothing else.